### PR TITLE
Cherry picking

### PR DESCRIPTION
**Pull request by teolemon**
Cherry picking

### CODE REVIEW

File: po/common/fr.po [hunk 1]
```diff
@@ -4401,7 +4401,7 @@ msgstr "Les photos du produits peuvent aussi être fournies séparément, via la
 
 msgctxt "attribute_group_labels_name"
 msgid "Labels"
-msgstr "Labels"
+msgstr "Étiquettes"
```
Comment (stephanegigandet) [inline po/common/fr.po:4404]: This one should be changed.

File: po/common/fr.po [hunk 2]
```diff
@@ -4962,26 +4954,79 @@ msgctxt "delete_all_preferences"
...
+msgid "Products that do not match your preferences"
+msgstr "Produits qui ne correspondent pas à vos préférences"
+
+msgctxt "products_match_unknown"
+msgid "Products for which we currently miss data to determine if they match your preferences"
+msgstr "Produits pour lesquels nous manquons actuellement de données pour déterminer s'ils correspondent à vos préférences"
+
+msgctxt "forest_footprint"
+msgid "Forest footprint"
+msgstr "Empreinte forestière"
```
Comment (stephanegigandet) [inline po/common/fr.po:4979]: Empreinte forêt

File: po/common/fr.po [hunk 3]
```diff
@@ -4962,26 +4954,79 @@ msgctxt "delete_all_preferences"
...
+msgid "Ingredients requiring soy"
+msgstr "Ingrédients nécessitant du soja"
+
+msgctxt "type"
+msgid "Type"
+msgstr "Type"
+
+msgctxt "processing_factor"
+msgid "Processing factor"
+msgstr "Facteur de traitement"
```
Comment (stephanegigandet) [inline po/common/fr.po:4991]: Facteur de transformation

File: po/common/en.po [hunk 4]
```diff
@@ -5130,68 +5130,3 @@ msgstr "Products that do not match your preferences"
 msgctxt "products_match_unknown"
 msgid "Products for which we currently miss data to determine if they match your preferences"
 msgstr "Products for which we currently miss data to determine if they match your preferences"
-
-msgctxt "forest_footprint"
```
Comment (stephanegigandet) [inline po/common/en.po:5134]: why are those strings removed?

Comment (teolemon) [reply po/common/en.po:5134]: because of the discrepancies between common.pot and en.po

File: po/common/fr.po (hunk 3 above)
Comment (teolemon) [inline po/common/fr.po:4991]:
```suggestion
msgstr "Facteur de transformation"
```

File: po/common/fr.po (hunk 2 above)
Comment (teolemon) [inline po/common/fr.po:4979]:
```suggestion
msgstr "Empreinte forêt"
```

**Pull request closed without merging**